lib/collections.js

```diff
@@ -157,13 +157,6 @@ var factory = module.exports = function (request) {
 
             if (query && typeof query === 'object') {
                 requestObject.endpoint = getPath(collection);
-
-                var qs = [];
-                Object.keys(query, function (key, value) {
-                    qs.push(key + ':' + value);
-                });
-
-                requestObject.qs = { q: qs.join(';') };
             }
```

Comment:
In my testing you'd want to have `requestObject.qs = query;` here so that you can pass parameters like `labelSelector`, etc. through to the underlying endpoint.